mirna_explorer/repository/create_database.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python3
+
+import os
+import sqlite3
+
+data_folder = "../../../mirna_target_prediction/data/"
+
+required_files = ["raw/UTR_Sequences.txt",
+				  "processed/mirna_id_dictionary.tsv",
+				  "processed/gene_id_dictionary.tsv",
+				  "processed/scored_interactions_processed.tsv"]
+
+for filename in required_files:
+	path = data_folder + filename
+	if not os.path.isfile(path):
+		print(f'error: file not found: \'{path}\', aborting')
```

Comment:
Remove the backslashes found in the string printed. To do so, replace the opening single quote to double quotes, i.e. f"error: file not found: '{path}', aborting"

Reply:
I kept the outer single quotes and replaced the escaped single quotes with double quotes.